frontend/src/component/feature/FeatureStrategy/FeatureStrategyConstraints/ValueList.tsx

```diff
@@ -14,16 +14,26 @@ const StyledList = styled('ul')({
     display: 'contents',
 });
 
+export const baseChipStyles = (theme: Theme) => ({
```

Comment:
I'm not entirely convinced by this extraction. Happy to revert it if you think it's useful. It's only in two places so it's ... actually a step **before** I think I'd normally like to extract it. But at the same time, it's also something that should change together 🤷🏼